app/forms/idv/api_image_upload_form.rb

```diff
@@ -458,14 +458,16 @@ def store_failed_images(client_response, doc_pii_response)
             front_image_fingerprint: failed_front_fingerprint,
             back_image_fingerprint: failed_back_fingerprint,
             doc_auth_success: client_response.doc_auth_success?,
-            selfie_success: client_response.selfie_success,
+            selfie_status: client_response.respond_to?(:selfie_status) ?
+             client_response.selfie_status : :not_processed,
```

Comment:
nit: this only indents by 1? I'd do 2
```suggestion
            selfie_status: client_response.respond_to?(:selfie_status) ?
              client_response.selfie_status : :not_processed,
```